pyflakes/checker.py

```diff
@@ -992,7 +992,10 @@ def checkDeadScopes(self):
 
             if all_binding:
                 all_names = set(all_binding.names)
-                undefined = all_names.difference(scope)
+                undefined = [
```

Comment:
I wonder if we can drop `all_names` or if the usage on L1026 is really that much faster just because it's a set in practice on real code-bases

Reply:
Yeah I thought about this, but I don't really have a good way to profile this.  I imagine it's not a significant difference to keep the original value as a list and just do `in` against that (however it does appear to change this from `O(N) + O(M)` to  `O(N * M)` where N is the number of imports and M is the number of items in `__all__`

Probably ok to just leave it since it keeps this change a little bit more minimal